### PR TITLE
Refactor filter descriptions

### PR DESCRIPTION
**Pull request by MaxPostema**
Set of changes coming from #5

@connoratrug:
- [x] Suggest using 'collapsed' , with required false and default of false
- [x] Maybe add a collapsable prop ( defaults to true ? )
- [x] Maybe add a bulk selectable ( default true ?) or is this part of the Type being passed ?
- [ ] What a about an id or key for when two options have the same value but a differ in text

@fdlk: 
- [ ] If we want a prop to control the state, we'll also need to fire an event when it changes. 

#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

**Comment by MaxPostema** [comment]
@connoratrug 
> What a about an id or key for when two options have the same value but a differ in text
I don't think this is an issue. The filter will return the list of selected values of any datatype. Having the same value is not useful at all, for anyone.

options: [{ value: true, text: 'Male' }, { value: true, text: 'Ape' }]
Will return [true] or [] and will toggle both toggles simultaneous.

options: [{ value:'chair', text: 'Chair' }, { value:'table', text: 'Table' }]
Is a better use case. ['chair', 'table'] is returned after selecting both options

I think using a boolean in the example is confusing the issue
options: [{ value:true, text: 'Yes' }, { value:false, text: 'No' }]
Makes no sense to be implemented as checkbox. We need a radio button filter for this.
Fixed the sample to be more clear.

**Comment by MaxPostema** [comment]
@fdlk 
> If we want a prop to control the state, we'll also need to fire an event when it changes.

There is a `input` event in `v-model="selections"` you can bind to if you split the shorthand. I think that would be a nice clean way to handle the situation.

File: README.md [hunk 1]
```diff
@@ -34,11 +34,12 @@ Every filter will have the following global properties:
 
 | name                 | required | default value | description | 
 |----------------------|----------|---------------|-------------|
-| name                 | true     | *nvt*         | string: name and id of the filter |
+| name                 | true     | -             | string: name and id of the filter |
```
Comment (connoratrug) [inline README.md]: why is '-' the default name ? should it just be an empty column ?

File: README.md [hunk 2]
```diff
@@ -34,11 +34,12 @@ Every filter will have the following global properties:
...
 | label                | true     |               | string: Name of the filter as used by the UI |
 | description          | false    |               | Descriptive text that is used by the UI |
-| initiallyCollapsed   | false    | true          | boolean: Open/Closed state of the card that holds the filter |
-| type                 | true     | *nvt*         | Vue Object that renders the filter |
+| collapsed            | false    | true          | boolean: Open/Closed state of the card that holds the filter |
```
Comment (connoratrug) [inline README.md]: i would argue the default should be 'false'

**Pull request merged**